programs/ProgressBarDemo.hs

```diff
@@ -48,13 +48,65 @@ drawUI p = [ui]
+              overrideAttr P.progressIncompleteAttr cToDoAttr1 
+              $ bar' (_x p) ('━', '━')
+      cBar2 = overrideAttr P.progressCompleteAttr cDoneAttr2 $
+              overrideAttr P.progressIncompleteAttr cToDoAttr2
+              $ bar' (_y p) ('▰', '▰')
+      cBar3 = overrideAttr P.progressCompleteAttr cDoneAttr $
+              overrideAttr P.progressIncompleteAttr cToDoAttr 
+              $ bar' (_z p) ('━', '─')
+      cBar4 = overrideAttr P.progressCompleteAttr cDoneAttr $
+              overrideAttr P.progressIncompleteAttr cToDoAttr 
+              $ bar' (_x p) ('▰', '▱')
+      cBar5 = overrideAttr P.progressCompleteAttr cDoneAttr $
+              overrideAttr P.progressIncompleteAttr cToDoAttr 
+              $ bar' (_y p) ('━', '=')
+      cBar6 = overrideAttr P.progressCompleteAttr cDoneAttr $
+              overrideAttr P.progressIncompleteAttr cToDoAttr 
+              $ bar' (_z p) ('█', '▁')
+      cBar7 = overrideAttr P.progressCompleteAttr cDoneAttr $
+              overrideAttr P.progressIncompleteAttr cToDoAttr 
+              $ bar' (_x p) ('■', '□')
+      cBar8 = overrideAttr P.progressCompleteAttr cDoneAttr $
+              overrideAttr P.progressIncompleteAttr cToDoAttr 
+              $ bar' (_y p) ('▪', '▫')
+      cBar9 = overrideAttr P.progressCompleteAttr cDoneAttr $
+              overrideAttr P.progressIncompleteAttr cToDoAttr 
+              $ bar' (_z p) ('▸', '▹')
+      cBar10 = overrideAttr P.progressCompleteAttr cDoneAttr $
+               overrideAttr P.progressIncompleteAttr cToDoAttr 
+               $ bar' (_x p) ('⣿', '⠶')
+      cBar11 = overrideAttr P.progressCompleteAttr cDoneAttr $
+               overrideAttr P.progressIncompleteAttr cToDoAttr 
+               $ bar' (_y p) ('|', '=')
+      cBar12 = overrideAttr P.progressCompleteAttr cDoneAttr $
+               overrideAttr P.progressIncompleteAttr cToDoAttr 
+               $ bar' (_z p) ('●', '⊙')
+      lbl c = if _showLabel p 
+              then Just $ " " ++ (show $ fromEnum $ c * 100) ++ " " 
+              else Nothing
       bar v = P.progressBar (lbl v) v
+      bar' v chars = P.progressBar' (lbl v) v chars
       ui = (str "X: " <+> xBar) <=>
            (str "Y: " <+> yBar) <=>
            (str "Z: " <+> zBar) <=>
+           (str "X: " <+> cBar1) <=>
+           (str "Y: " <+> cBar2) <=>
+           (str "Z: " <+> cBar3) <=>
+           (str "X: " <+> cBar4) <=>
+           (str "Y: " <+> cBar5) <=>
+           (str "Z: " <+> cBar6) <=>
+           (str "X: " <+> cBar7) <=>
+           (str "Y: " <+> cBar8) <=>
+           (str "Z: " <+> cBar9) <=>
+           (str "X: " <+> cBar10) <=>
+           (str "Y: " <+> cBar11) <=>
+           (str "Z: " <+> cBar12) <=>
            str "" <=>
-           str "Hit 'x', 'y', or 'z' to advance progress, or 'q' to quit"
+           str "Hit 'x', 'y', or 'z' to advance progress, 'r' to revert values, 'cmd + r' to reset values or 'q' to quit"
```

Comment:
This help text was modified, but it doesn't mention the new `t` binding that was also added.